targets/zephyr/Makefile.travis

```diff
@@ -27,12 +27,15 @@ install-apt-get-deps:
 
 # Install Zephyr SDK.
 install-zephyr-sdk:
-	wget https://github.com/zephyrproject-rtos/meta-zephyr-sdk/releases/download/0.9.1/zephyr-sdk-0.9.1-setup.run -O ../zephyr-sdk-0.9.1-setup.run
-	sh ../zephyr-sdk-0.9.1-setup.run -- -y -d $(CURDIR)/../zephyr-sdk-0.9.1
+	wget https://github.com/zephyrproject-rtos/meta-zephyr-sdk/releases/download/0.9.2/zephyr-sdk-0.9.2-setup.run -O ../zephyr-sdk-0.9.2-setup.run
+	sh ../zephyr-sdk-0.9.2-setup.run -- -y -d $(CURDIR)/../zephyr-sdk-0.9.2
+	wget https://cmake.org/files/v3.8/cmake-3.8.2-Linux-x86_64.sh
+	sudo sh cmake-3.8.2-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir
+	cmake --version
 
 # Fetch Zephyr Project repository and install python dependencies.
 install-zephyr: install-apt-get-deps
-	git clone https://github.com/zephyrproject-rtos/zephyr.git ../zephyr -b zephyr-v1.9.1
+	git clone https://github.com/zephyrproject-rtos/zephyr.git ../zephyr -b v1.10-branch
```

Comment:
Will this branch live forever? IMHO it will be better to use the release tag and not the branch.

Reply:
This branch would transition thru 1.10.x point releases, and the idea was to check that JrS remains buildable with them, but as you wish, switching to the tag.